fedireads/status.py

```diff
@@ -23,11 +23,8 @@ def create_review_from_activity(author, activity):
     return review
 
 
-def create_review(user, possible_book, name, content, rating):
+def create_review(user, book, name, content, rating):
```

Comment:
This seems sensible to me but it will necessitate updating `create_review_from_activity` (and maybe others)

Reply:
Oops, I thought I checked for other callers. I'll check again and fix. Thanks!

Reply:
I think it was just that one.